lib/kafo/answer_file.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+
+module Kafo
+  class AnswerFile
+
+    attr_reader :answers, :filename, :version
+
+    def initialize(answer_filename, version: 1)
+      @filename = answer_filename
+      @version = version.nil? ? 1 : version
+
+      begin
+        @answers = YAML.load_file(@filename)
+      rescue Errno::ENOENT
+        KafoConfigure.exit(:no_answer_filename) do
+          puts "No answer file found at #{@filename}"
+        end
+      end
+
+      validate
+    end
+
+    def filename
+      @filename
+    end
+
+    def puppet_classes
```

Comment:
Note that previously, and still in `configuration.rb` the nomenclature uses modules and not classes. However, what is actually declared are classes in the answers file so I chose to go with a nomenclature that is more accurate for this new entity.